Tensile/Common.py

```diff
@@ -783,7 +783,7 @@ def getArchitectureName(gfxName):
     # StaggerUStride will be internally increased so it is an integer multiple of DepthU*BpeAB.
     # (the implementation requires this - the unroll iteration accesses data in steps of
     # DepthU*BPE
-    "StaggerUStride":               [16,32,64,128,256,512,1024],
+    "StaggerUStride":        [0,16,32,64,128,256,512,1024],
```

Comment:
Please put some comment here. Like 0 is for StaggerU=0 or something.

Reply:
Done, Thanks.